<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>391</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>幸运的人儿</string>
   </property>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;华文琥珀&quot;;
color: rgb(255, 11, 3);</string>
    </property>
    <property name="text">
     <string>一等奖</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;华文琥珀&quot;;
color:rgb(255, 162, 0)</string>
    </property>
    <property name="text">
     <string>二等奖</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;华文琥珀&quot;;
color:rgb(48, 62, 255)</string>
    </property>
    <property name="text">
     <string>三等奖</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文仿宋&quot;;</string>
    </property>
    <property name="text">
     <string>是谁呢QWQ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文仿宋&quot;;</string>
    </property>
    <property name="text">
     <string>是谁呢QAQ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文仿宋&quot;;</string>
    </property>
    <property name="text">
     <string>是谁呢QWQ</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
